protos/flyteidl/core/execution.proto

```diff
@@ -18,6 +18,7 @@ message WorkflowExecution {
         FAILED = 6;
         ABORTED = 7;
         TIMED_OUT = 8;
+        RECOVERED = 9;
```

Comment:
What does this state represent? so when we create a new execution, it'll go to RECOVERED instead of QUEUED or before/after QUEUED? I'm not sure we actually need a state to track it... mind clarifying how that will work?

Reply:
removed, thanks for pointing out